src/test/java/org/example/DeliveryToFunctionality.java

```diff
@@ -0,0 +1,127 @@
+package org.example;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.time.Duration;
+import java.util.List;
+
+public class DeliveryToFunctionality {
+
+    public WebDriver webDriver;
+    public String poland = "Poland";
+    public String zip = "90210";
+
+    @BeforeMethod
+    public void setup() {
+        WebDriverManager.chromedriver().setup();
+        // System.setProperty("webdriver.chrome.driver", "E:\\it\\java\\web_driver_test\\src\\test\\resources\\webdriver\\chromedriver.exe");
```

Comment:
Why did you leave this comment

Reply:
This comment I leave for myself)))) just to have an example how it could be. Don't pay attention to it

Reply:
Yes, it is better to remove it. On real projects, we do not do like this